src/main/resources/templates/c_plus_plus/readme

```diff
@@ -0,0 +1,15 @@
+1. [task][CMake runs correctly](TestConfigure)
+2. [task][Your code compiles](CompileSort)
+3. [task][Code is formatted with clang-format](clang-format(sort.cpp))
```

Comment:
Why is this a test instead of SCA-feedback that subtracts points in case of violations?

Reply:
These were extracted from an exercise I created for a programming class - I was not familiar with Artemis when creating then ;)

Reply:
I'll leave it out for now, and add SCA support in a follow-up PR.